test/unit/canvas_static.js

```diff
@@ -192,8 +192,6 @@
     var canvas = new fabric.StaticCanvas();
     assert.ok('backgroundColor' in canvas);
     assert.ok('overlayColor' in canvas);
-    assert.ok('backgroundImage' in canvas);
-    assert.ok('overlayImage' in canvas);
```

Comment:
do you know why those are not true anymore?

Reply:
maybe `canvas.hasOwnProperty` would work? how do wecheck we don't accidentally remove stuff from the classes? this was meant to do this only

Reply:
just making a test....

Reply:
No need
I wrote in the description that I changed the type from null to undefined.
So there was no sense in setting these field to undefined in default props so I removed them.
That is all.

Reply:
Not sure.
But that is the point of declare, isn't it?
Without declare it is defined as void 0.
But then we have problems in initialization.
That can be fixed of coursr but will require a lot if refactoring